rdr_service/cron_prod.yaml

```diff
@@ -104,3 +104,8 @@ cron:
   timezone: America/New_York
   schedule: every day 01:00
   target: offline
+- description: Genomic GC Appointment GRoR Revoked Notification (Daily)
```

Comment:
I know in the ticket there is verbiage about GROR being revoked but can we go with the naming convention of something like "GenomicAppointmentGRORChanged" and have that follow suite in the namings of this PR.